src/state/apis/zapper/zapperApi.ts

```diff
@@ -0,0 +1,63 @@
+import { createApi } from '@reduxjs/toolkit/dist/query/react'
+import type { AssetId } from '@shapeshiftoss/caip'
+import { ethChainId, toAssetId } from '@shapeshiftoss/caip'
+import { getConfig } from 'config'
+import qs from 'qs'
+import { BASE_RTK_CREATE_API_CONFIG } from 'state/apis/const'
+
+import { chainIdToZapperNetwork, createApiClient, zapperNetworkToChainId } from './client'
+
+const ZAPPER_BASE_URL = 'https://api.zapper.xyz'
+
+const authorization = `Basic ${Buffer.from(
+  `${getConfig().REACT_APP_ZAPPER_API_KEY}:`,
+  'binary',
+).toString('base64')}`
+
+const headers = {
+  accept: 'application/json',
+  authorization,
+}
+
+const zapperClient = createApiClient(ZAPPER_BASE_URL)
+
+type GetAppBalancesInput = {}
+type GetAppBalancesOutput = AssetId[]
+
+// https://docs.zapper.xyz/docs/apis/getting-started
+export const zapperApi = createApi({
+  ...BASE_RTK_CREATE_API_CONFIG,
+  reducerPath: 'zapperApi',
+  endpoints: build => ({
+    getZapperUniV2PoolAssetIds: build.query<GetAppBalancesOutput, GetAppBalancesInput>({
+      queryFn: async () => {
+        const evmNetworks = [chainIdToZapperNetwork(ethChainId)]
+
+        const zerionV2AppTokensData = await zapperClient.getV2AppTokens({
+          params: {
+            appSlug: 'uniswap-v2',
```

Comment:
maybe abstract to constant, or comment, or link to docs where this comes from and why it's univ2 pools only for now

Reply:
porque no los dos https://github.com/shapeshift/web/pull/4144/commits/f1445aa832fc9dfe9c236b26912eb922a0ec3f31